src/client/modules/counter/containers/Counter.jsx

```diff
@@ -11,6 +11,16 @@ import ADD_COUNTER from '../graphql/AddCounter.graphql';
 import COUNTER_SUBSCRIPTION from '../graphql/CounterSubscription.graphql';
 
 class Counter extends React.Component {
+  static defaultProps = {
+    loading: false,
+    subscribeToMore: () => {}
```

Comment:
Do we gain anything with defining this for every function?

Reply:
It is just a good practice to define `defaultProps`, especially in the case where the `PropTypes` is marked as required. If the default is not defined it could create problems down the line in the code where you see missing prop `jsx` errors.

Reply:
@mairh Why would anyone want to set `defaultProps` on `required` props? I can understand why they can be needed on optional props in some cases, but I'm totally don't get why use them on `required` props?

Reply:
@mairh When defaultProps are used with require props, the typechecking is lost (e.g. React no longer checks if required property was provided), since props typechecking happens after applying defaultProps. See here:
https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values

Reply:
So shall we just use it for non required props only?

Reply:
@mairh I don't think it should be used for any props. Only if you want to have default value for optional prop.